ethcore/types/src/snapshot.rs

```diff
@@ -39,31 +41,53 @@ pub enum Snapshotting {
 }
 
 /// A progress indicator for snapshots.
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub struct Progress {
 	/// Number of accounts processed so far
-	pub accounts: AtomicUsize,
+	accounts: AtomicUsize,
```

Comment:
with this many fields, it's almost certainly better to use a single parking lot mutex wrapping a single struct containing all this data. maybe beyond the scope

Reply:
Good idea, I'll make the change in a diff PR as it gets a bit noisy.